api/serializers/student.py

```diff
@@ -25,3 +29,32 @@ class StudentReadSerializer(serializers.ModelSerializer[Student]):
     class Meta:
         model = Student
         exclude = ("children",)
+
+
+class PublicStudentSerializer(serializers.ModelSerializer[Student]):
```

Comment:
add a short docstring explaining what "public" is in this context
(None of the information is actually public, so I guess you mean "available to all coordinators" or "to all users of the dashboard").

Reply:
I mean "internal" api is for bot and "public" is for tooljet :)

Reply:
But added comment